kpi/deployment_backends/kobocat_backend.py

```diff
@@ -577,6 +591,73 @@ def get_submissions(self, requesting_user_id,
             )
         return submissions
 
+    @staticmethod
+    def _get_updated_instance_id():
+        uuid_ = str(uuid.uuid4())
+        return uuid_, f'uuid:{uuid_}'
+
+    @staticmethod
+    def _get_updated_datetime():
+        """
+        Matching the OpenRosa datetime formatting
+        """
+        return datetime.now(tz=pytz.UTC).isoformat('T', 'milliseconds')
+
+    def duplicate_submission(
+        self, requesting_user_id, instance_id, **kwargs
+    ):
+        """
+        Dupicates a single submission through `kpi`, proxied through `kc`. The
+        submission with the given `pk` is duplicated and the `start`, `end` and
+        `instanceID` parameters of the submission are reset before being posted
+        to `kc`.
+
+        Args:
+            requesting_user_id (int)
+            instance_id (int)
+            kwargs (dict): passed to validation
```

Comment:
Correct - only `instance_ids` is passed on. Should the doc string be changed to indicate this - or even remove `kwargs` as a parameter since `instance_ids` is set within the method and not passed from outside?

Reply:
I'd say remove `kwargs`